### PR TITLE
Big restructuring based on docker-compose deployments

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: ansible/roles/docker/templates/service.j2 [hunk 1]
```diff
@@ -12,10 +12,13 @@ Type=notify
 TimeoutStartSec=0
 Delegate=yes
 KillMode=process
-LimitNOFILE=infinity
+LimitNOFILE=1048576
```
Comment (bkcsoft) [inline ansible/roles/docker/templates/service.j2:15]: Do we know the kernel setting?

**Review by bkcsoft — COMMENTED** [review]
otherwise LGTM

File: ansible/roles/docker/templates/service.j2 (hunk 1 above)
Comment (tboerger) [reply ansible/roles/docker/templates/service.j2:15]: I have not checked the kernel setting, this should be the default setting from the docker package

**Comment by lunny** [comment]
Trusted LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**